src/api/libcellml/generatorprofile.h

```diff
@@ -18,6 +18,8 @@ limitations under the License.
 
 #include "libcellml/exportdefinitions.h"
 
+#include <string>
```

Comment:
This is not needed since it's already included by `libcellml/exportdefinitions.h`.

Reply:
Sorry, I guess I figured I was having a problem with std::string in the Python bindings.  Whilst it is actually std::shared_ptr.